src/WebJobs.Script/Binding/BlobBinding.cs

```diff
@@ -53,12 +54,14 @@ public BlobBinding(ScriptHostConfiguration config, BlobBindingMetadata metadata,
             await BindStreamAsync(context.Value, Access, context.Binder, runtimeContext);
         }
 
-        public override Collection<CustomAttributeBuilder> GetCustomAttributes()
+        public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type parameterType)
         {
             Collection<CustomAttributeBuilder> attributes = new Collection<CustomAttributeBuilder>();
 
+            FileAccess access = GetAttributeAccess(parameterType);
```

Comment:
It would still be helpful to have the parameter type passed in as it might be helpful in other scenarios as well. I wondered about the possible mismatch, but the alternatives didn't sound any better. We're not (currently) aware of the target type when the metadata is created.